lib/api/element-commands/click.js

```diff
@@ -40,17 +56,59 @@ const BaseElementCommand = require('./_baseElementCommand.js');
  * @param {string} [using] The locator strategy to use. See [W3C Webdriver - locator strategies](https://www.w3.org/TR/webdriver/#locator-strategies)
  * @param {string} selector The CSS/Xpath selector used to locate the element.
  * @param {function} [callback] Optional callback function to be called when the command finishes.
+ * @param {string} [message] Optional message to be shown in the output in case of success
  * @api protocol.elementinteraction
  * @link /#dfn-element-click
  */
 class ClickElement extends BaseElementCommand {
-  get extraArgsCount() {
-    return 0;
+  get minArgsCount() {
+    return 1;
+  } 
+
+  get maxArgsCount() {
+    return 4;
+  } 
+
+  validateArgsCount() {
+    if (this.args.length < this.minArgsCount || this.args.length > this.maxArgsCount) {
+      throw new Error(`${this.commandName} method expects betweeen ${(this.minArgsCount)} `+
+        `to  ${(this.maxArgsCount)} arguments - ${this.args.length} given.`);
+    }
+
+    if (LocateStrategy.isValid(this.args[0]) && (Utils.isString(this.args[1]) || ElementCommand.isSelectorObject(this.args[1]))) {
+      this.setStrategyFromArgs();
+    }
   }
 
   get elementProtocolAction() {
     return 'clickElement';
   }
+  
+  async successHandler(result){
+    const {reporter} = this.client;
+    const {message} = this;
+    const passed = true;
+    const err = {};
+
+    const runner = new AssertionRunner({
```

Comment:
I don't think we should use AssertionRunner here. \`click\` command is not an assert type command. Maybe we can achieve the same goal by making changes in `_baseElementCommand.js`

Reply:
@beatfactor what do you think about this change?

Reply:
@beatfactor what do you think about this change?

Reply:
Yes, you're right. AssertionRunner shouldn't be used here.